### ISSUE TITLE
How to run?

### ISSUE THREAD

**Issue by tonypartridge** [renatomarinho/laravel-gitscrum#43]
Hey guys,

I must be missing something here, how do we get Gitscrum to boot? I've run everything perfectly. But I see no  way of running Gitscrum? Do you have to use docker? If so whats the default command to boot it?

**Comment by yinx** [comment]
You can use docker if you want or you can use a virtual environment like valet or homestead.
If you want to use docker, there is a docker-compose file that should boot the application.
First make sure you have docker installed ( https://www.docker.com/ ).
Next make sure the docker services are running.
Now you can try running 'docker-compose up -d' in the root folder of Gitscrum.
Make sure port 80 is free, or if something is already using port 80 you can change which port the docker container uses in the docker-compose.yml file in the web section.
This should boot up a docker container with the gitscrum application.
You can then access it at localhost:8000 or whichever port you set in the docker-compose.yml file.

Hope that helps.

**Comment by tonypartridge** [comment]
@yinx I suspect it's down to my LAMP stack, but getting a 500 error, usually turning out debug mode would throw the error for me. But nothing is working, do you have any suggestions, this is on the wizard step 1.

**Comment by yinx** [comment]
@tonypartridge So you got it running but you're getting an error on a certain page right?
Did you fill in the github app credentials and the credentials to the database?
The explanation for this should be in the readme file.

**Comment by tonypartridge** [comment]
@yinx, yep all in processed past Github and authorised. I suspect it's a php fatal error being thrown as an application 500 error. Likely down to the fact I'm on php7 with mysql56 / apache whereas your docker is mysql57 and nginx.

**Comment by renatomarinho** [comment]
Try use the repos Docker-GitScrum ( https://github.com/renatomarinho/docker-gitscrum )

**Comment by mhamzas** [comment]
I am also getting this 500 error. I am using php 7. 
I've tried enabling debug config but not getting it. Any suggestions ?

**Comment by tonypartridge** [comment]
I didn't get much further.. Client jobs and bills taking priority I'm afraid.

**Comment by mhamzas** [comment]
It was my mistake. I am using vestaCP and I had openbase_dir error.
So I've made the changes on my httpd config file and it worked !

Thanks for your response.
